images/build/GitHubActions.Dockerfile

```diff
@@ -0,0 +1,165 @@
+# Folders in the image which we use to build this image itself
```

Comment:
nit: use directories instead of folders may be?

Reply:
This is just  a comment :-)